src/server/check-answers/index.js

```diff
@@ -93,6 +93,49 @@ export class SubmitPageController extends QuestionPageController {
     return super.handleGet(req, h)
   }
 
+  async _handleToCaeManagementApi(req, h) {
+    const state = new StateManager(req)
+    const applicationState = state.toState()
+
+    const application = ApplicationModel.fromState(applicationState)
+
+    const { message } = await application.send()
+    req.yar.set('applicationReference', message)
+
+    return super.handlePost(req, h)
+  }
+
+  async handlePost(req, h) {
+    const payload = /** @type {ConfirmationPayload & NextPage} */ (req.payload)
+    const confirmation = new ConfirmationAnswer(payload)
+    const { isValid: isValidPage } = confirmation.validate()
+    const state = new StateManager(req)
+    const applicationState = state.toState()
+
+    const application = ApplicationModel.fromState(applicationState)
+
+    const { isValid: isValidApplication } = application.validate()
+
+    if (isValidPage && isValidApplication) {
+      if (
+        config.get('featureFlags').sendToCaseManagement ||
+        (req.query['case-management-api'] === 'true' &&
+          !config.get('featureFlags').sendToCaseManagement)
```

Comment:
I think this line is redundant.  On this side of the `or` expression, we know that `config.get('featureFlags').sendToCaseManagement` is at least falsey already.